src/state/slices/expandedPostSlice.ts

```diff
@@ -0,0 +1,29 @@
+import { PayloadAction, createSlice } from "@reduxjs/toolkit";
+import { Post } from "./postsSlice";
+
+const initialState: Post = {
+  displayName: "",
+  isLikedByCurrentUser: false,
+  numberOfLikes: 0,
+  postId: 0,
+  textContent: "",
+  timestamp: "",
+  username: "",
+};
+
+export const postSlice = createSlice({
+  name: "post",
+  initialState,
+  reducers: {
+    setPost: (_, action: PayloadAction<Post>) => {
```

Comment:
Can we please rename this reducer to "setExpandedPost"?

Reply:
This has been updated